app/controllers/projects_controller.rb

```diff
@@ -18,9 +18,33 @@ class ProjectsController < ApplicationController
 
   helper_method :repo_data
 
+  # These are the only allowed values for "sort" (if a value is provided)
+  ALLOWED_SORT =
+    %w(
+      id name achieved_passing_at badge_percentage
+      homepage_url repo_url updated_at user_id created_at
+    ).freeze
+
+  # If a valid "sort" parameter is provided, sort @projects in "sort_direction"
+  # rubocop:disable Metrics/AbcSize, Style/ConditionalAssignment
+  def sort_projects
+    # Sort, if there is a requested order (otherwise use default created_at)
+    if params[:sort].present? && ALLOWED_SORT.include?(params[:sort])
+      if params[:sort_direction] == 'desc' # descending
+        sort_direction = ' desc'
+      else
+        sort_direction = ' asc' # default is ascending
+      end
```

Comment:
Good point.  I had to tweak this a bit to make rubocop happy, but it's in there.